great_expectations/render/renderer/datasource_new_notebook_renderer.py

```diff
@@ -115,9 +116,12 @@ def render(self) -> nbformat.NotebookNode:
         self._add_docs_cell()
         if self.datasource_type == DatasourceTypes.SQL:
             self._add_sql_credentials_cell()
-        self._add_template_cell()
-        self._add_test_yaml_cells()
-        self._add_save_datasource_cell()
+
+        lint = has_black_formatter()
+        self._add_template_cell(lint)
+        self._add_test_yaml_cells(lint)
+        self._add_save_datasource_cell(lint)
```

Comment:
Rather than have the `lint` arg in these helper methods default to `True`, conditionally determine the value based on `black`'s presence